deploy/kubernetes/base/node.yaml

```diff
@@ -56,7 +54,6 @@ spec:
             failureThreshold: 5
         - name: csi-driver-registrar
           image: quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
-          imagePullPolicy: Always
```

Comment:
Same above

Reply:
imagePullPolicy was not added in helm (so no need to change).